test/testinput/bufr_ncep_adpupa_nc002003.yaml

```diff
@@ -0,0 +1,150 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
```

Comment:
Change the copyrights to 2022

Reply:
@nicholasesposito changed the year from 2020 to 2022, thanks.